readme.md

```diff
@@ -0,0 +1,225 @@
+# An Open Source Multilayer SMA Coaxial to Microstrip Transition
```

Comment:
Titles for posting to other sites

- How to create low-cost HF design
- A poor man's HF simulation
- The powerful HF simulator, which is missing a UI
- The powerful HF simulator, which is missing a community